### PR TITLE
doc: Add Generic Node Sensor Edition

### PR DESCRIPTION
**Pull request by pradeeka7**
#### Summary

Document the process of registering the Generic Node Sensor Edition with The Things Stack. Link to the issue: https://github.com/TheThingsIndustries/lorawan-stack-docs/issues/988

#### Screenshots

image

#### Changes

- New document
- ...

#### Notes for Reviewers

@nejraselimovic @benolayinka Can you review?
I am unable to provide REAL screen captures for certain steps (ie. screenshots with JoinEUI all zeros etc) due to the absence of a QR code on my Generic Node. 
...

#### Checklist

- [X] Scope: The referenced issue is addressed, there are no unrelated changes.
- [X] Run Locally: Verified that the docs build using `make server`, posted screenshots, verified external links. Test with `HUGO_PARAMS_SEARCH_ENABLED=true` if style changes will affect the search bar.
- [X] New Features Marked: Documentation for new features is marked using the `new-in-version` shortcode, according to the guidelines in [CONTRIBUTING](CONTRIBUTING.md).
- [X] Style Guidelines: Documentation obeys style guidelines in [CONTRIBUTING](CONTRIBUTING.md).
- [X] Commits: Commit messages follow guidelines in [CONTRIBUTING](CONTRIBUTING.md), there are no fixup commits left.

### CODE REVIEW

File: doc/content/devices/generic-node-se/_index.md [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+- In {{% tts %}} **Console**, choose the [application](/integrations/adding-applications/) to which you want to add the Generic Node.
+- Select the **Register end device** button.
+
+{{< figure src="application-overview.png" alt="Application overview" >}}
+
+- In the **Register end device** page select the **Scan end device QR code** button.
+
+{{< figure src="register-end-device.png" alt="Register end device" >}}
+
+- The camera will start up and be ready to scan the QR code. To begin, bring the QR code for your device close to the camera.
```
Comment (nejraselimovic) [inline doc/content/devices/generic-node-se/_index.md:27]:
```suggestion
- The camera will start up and be ready to scan the QR code. To scan the device, bring the QR code close to the camera.
```

File: doc/content/devices/generic-node-se/_index.md [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+
+{{< figure src="waiting-for-camera.png" alt="Waiting for camera" >}}
+
+- A window with information from the QR code should appear on your screen, displaying the **Claim authentication code**, **JoinEUI**, **DevEUI**, and **Brand** for your device. If the data displayed is correct, select the **Apply** button.
```
Comment (nejraselimovic) [inline doc/content/devices/generic-node-se/_index.md:31]:
```suggestion
- A window with information scanned from the QR code (like **Claim authentication code**, **JoinEUI**, **DevEUI** and **Brand**) should appear on your screen. If the data displayed is correct, select the **Apply** button.
```

File: doc/content/devices/generic-node-se/_index.md [hunk 3]
```diff
@@ -0,0 +1,87 @@
...
+
+This section will provide guidance on how to begin using [Generic Node Sensor Edition](https://www.genericnode.com/docs/sensor-edition/) on {{% tts %}}.
+
+The Generic Node Sensor Edition comes with a pre-loaded application called [vanilla](https://www.genericnode.com/docs/applications/se-vanilla/) that showcases the device's functions and capabilities. This application utilizes a combination of the onboard components including the LEDs, button, buzzer, accelerometer, secure element, temperature sensor, and humidity sensor. The instructions on how to flash a new application onto the Generic Node SE can be found [here](https://www.genericnode.com/docs/getting-started/se-sw/env-setup/stm32cubeide/).
+
+# Onboarding Using the QR Code
+
+The Generic Node can be onboarded to {{% tts %}} using the QR code which is printed on the back of its enclosure.
+
+- Sign in to [{{% tts %}}](/getting-started/).
```
[the remark this answers is not in the document]
Comment (nejraselimovic) [reply doc/content/devices/generic-node-se/_index.md:17]: Also, remove `-` bullets from this and following steps.

File: doc/content/devices/generic-node-se/_index.md [hunk 4]
```diff
@@ -0,0 +1,87 @@
...
+
+{{< figure src="waiting-for-camera.png" alt="Waiting for camera" >}}
+
+- A window with information from the QR code should appear on your screen, displaying the **Claim authentication code**, **JoinEUI**, **DevEUI**, and **Brand** for your device. If the data displayed is correct, select the **Apply** button.
+
+{{< figure src="found-qr-data.png" alt="Found QR data" >}}
+
+To continue with the rest of the registration process you can choose either **LoRaWAN Device Repository** or **manual registration**.
+
+## Using the LoRaWAN Device Repository
```
Comment (nejraselimovic) [inline doc/content/devices/generic-node-se/_index.md:37]:
```suggestion
### Using the LoRaWAN Device Repository
```

File: doc/content/devices/generic-node-se/_index.md [hunk 5]
```diff
@@ -0,0 +1,87 @@
...
+  - **End device ID**: you can continue with the pre-filled value or use any other unique value.
+- Select the **Register end device** button.
+
+{{< figure src="dev-rep-2.png" alt="Settings for registration through device repository" >}}
+
+The device is now registered and you will be redirected to the **overview** page of the device.
+
+{{< figure src="gnse-after-register.png" alt="GNSE overview page" >}}
+
+## Manually
```
Comment (nejraselimovic) [inline doc/content/devices/generic-node-se/_index.md:63]:
```suggestion
### Manual Registration
```

File: doc/content/devices/generic-node-se/_index.md [hunk 6]
```diff
@@ -0,0 +1,87 @@
...
+
+- A window with information from the QR code should appear on your screen, displaying the **Claim authentication code**, **JoinEUI**, **DevEUI**, and **Brand** for your device. If the data displayed is correct, select the **Apply** button.
+
+{{< figure src="found-qr-data.png" alt="Found QR data" >}}
+
+To continue with the rest of the registration process you can choose either **LoRaWAN Device Repository** or **manual registration**.
+
+## Using the LoRaWAN Device Repository
+
+The [Device repository](https://github.com/TheThingsNetwork/lorawan-devices) comprises over 600 end-device profiles, including the Generic Node Sensor Edition, which requires less information for registration with {{% tts %}}.
```
Comment (nejraselimovic) [inline doc/content/devices/generic-node-se/_index.md:39]:
```suggestion
The [LoRaWAN Device Repository](https://github.com/TheThingsNetwork/lorawan-devices) comprises over 600 end-device profiles, including the Generic Node Sensor Edition, which requires less information for registration with {{% tts %}}.
```

File: doc/content/devices/generic-node-se/_index.md [hunk 7]
```diff
@@ -0,0 +1,87 @@
...
+
+- In the **End device type** section, under **Input Method** select **Select the end device in the LoRaWAN device repository**. Then select/enter the following settings:
```
Comment (nejraselimovic) [inline doc/content/devices/generic-node-se/_index.md:41]: Remove the bullet

File: doc/content/devices/generic-node-se/_index.md [hunk 8]
```diff
@@ -0,0 +1,87 @@
...
+  - **End device brand**: The Things Industries
+  - **Model**: Generic Node (Sensor Edition)
+  - **Hardware Ver.**: 1.1
+  - **Firmware Ver.**: 1.1
+  - **Profile (Region)**: choose EU_863_870 for Europe or US_902_928 for US
+  - **Frequency plan**: choose the frequency plan appropriate for your region/country, for example, if you are live in Europe, choose **Europe 863-870 MHz (SF9 for RX2 - recommended)**.
```
Comment (nejraselimovic) [inline doc/content/devices/generic-node-se/_index.md:47]: For selected values use **bold** and for entered values use `highlight`

File: doc/content/devices/generic-node-se/_index.md [hunk 9]
```diff
@@ -0,0 +1,87 @@
...
+
+{{< figure src="dev-rep-1.png" alt="Settings for registration through device repository" >}}
+
+  - **JoinEUI**: pre-filled
+  - **DevEUI**: pre-filled (this is the secure element’s DevEUI)
+  - **AppKey**: select the **Generate** button.
+  - **End device ID**: you can continue with the pre-filled value or use any other unique value.
+- Select the **Register end device** button.
```
Comment (nejraselimovic) [inline doc/content/devices/generic-node-se/_index.md:55]: Remove bullet

File: doc/content/devices/generic-node-se/_index.md [hunk 10]
```diff
@@ -0,0 +1,87 @@
...
+
+{{< figure src="dev-rep-2.png" alt="Settings for registration through device repository" >}}
+
+The device is now registered and you will be redirected to the **overview** page of the device.
```
Comment (nejraselimovic) [inline doc/content/devices/generic-node-se/_index.md:59]:
```suggestion
The device is now registered and you will be redirected to the **Overview** page of the device.
```

File: doc/content/devices/generic-node-se/_index.md [hunk 11]
```diff
@@ -0,0 +1,87 @@
...
+- Under **Provisioning information** enter the following settings:
+  - **JoinEUI**: pre-filled
+  - **DevEUI**: pre-filled (this is the secure element’s DevEUI)
+  - **AppKey**: select the **Generate** button.
+  - **End device ID**: you can continue with the pre-filled value or use any other unique value.
+- Select the **Register end device** button.
+
+{{< figure src="manually-2.png" alt="Settings for manual registartion" >}}
+
+The device is now registered and you will be redirected to the **overview** page of the device.
```
Comment (nejraselimovic) [inline doc/content/devices/generic-node-se/_index.md:85]: Same comments as for the section above

**Review by nejraselimovic — CHANGES_REQUESTED** [review]

File: doc/content/devices/generic-node-se/_index.md [hunk 12]
```diff
@@ -0,0 +1,87 @@
...
+title: "Generic Node Sensor Edition"
+description: ""
+weight: 
+---
+
+{{< figure src="generic-node-se.png" alt="Generic Node Sensor Edition" class="float plain" >}}
+
+This section will provide guidance on how to begin using [Generic Node Sensor Edition](https://www.genericnode.com/docs/sensor-edition/) on {{% tts %}}.
+
+The Generic Node Sensor Edition comes with a pre-loaded application called [vanilla](https://www.genericnode.com/docs/applications/se-vanilla/) that showcases the device's functions and capabilities. This application utilizes a combination of the onboard components including the LEDs, button, buzzer, accelerometer, secure element, temperature sensor, and humidity sensor. The instructions on how to flash a new application onto the Generic Node SE can be found [here](https://www.genericnode.com/docs/getting-started/se-sw/env-setup/stm32cubeide/).
```
[the remark this answers is not in the document]
Comment (pradeeka7) [reply doc/content/devices/generic-node-se/_index.md:11]: Yes, indeed.

File: doc/content/devices/generic-node-se/_index.md [hunk 13]
```diff
@@ -0,0 +1,87 @@
+---
+title: "Generic Node Sensor Edition"
+description: ""
+weight: 
+---
+
+{{< figure src="generic-node-se.png" alt="Generic Node Sensor Edition" class="float plain" >}}
+
+This section will provide guidance on how to begin using [Generic Node Sensor Edition](https://www.genericnode.com/docs/sensor-edition/) on {{% tts %}}.
```
[the remark this answers is not in the document]
Comment (pradeeka7) [reply doc/content/devices/generic-node-se/_index.md:9]: Will do.

**Review by nejraselimovic — APPROVED** [review]

**Comment by nejraselimovic** [comment]
Nice job @pradeeka7!

**Pull request merged**